@here/harp-map-theme/resources/berlin_tilezen_base.json

```diff
@@ -464,7 +465,7 @@
                 "description": "primary-road-shield-5+, level > 10",
                 "when": [
                     "all",
-                    ["==", ["get", "$layer"], "roads"],
+                    ["==", ["get", "$layer"], "road_labels"],
```

Comment:
Is there anyway to have a fallback in case road_labels does not exist? The layer `road_labels` is an optional one that could be removed in the future, so, to be safe it would be better to have a logic such as: use road_labels if available and if not, use roads.
Is that even possible?

Reply:
As far as I know this is not possible b/c the when condition is only in scope of the current feature that is processed.
The only way I can imagine this to work is, if map compilation would add an additional attribute, e.g. `useForLabeling', to the roads and road_labels layer. That attribute we could check in the condition.